java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -21,11 +21,10 @@ public interface GenericClusterCommands {
      *
      * <p>The command will be routed to all primaries.
      *
-     * @remarks This function should only be used for single-response commands. Commands that don't
-     *     return response (such as <em>SUBSCRIBE</em>), or that return potentially more than a single
-     *     response (such as <em>XREAD</em>), or that change the client's behavior (such as entering
-     *     <em>pub</em>/<em>sub</em> mode on <em>RESP2</em> connections) shouldn't be called using
-     *     this function.
+     * <p>See <a
```

Comment:
```suggestion
     * @apiNote See <a
```
This could be more visual I think.

Please update thourhgout and re-run spotless (java linter)

Reply:
done